Jaeger tracing documentation updates (#2772)
wso2/docs-ei: en/micro-integrator/docs/setup/observability/setting-up-cloud-native-observability-in-kubernetes.md

This deployment installs Jaeger-Operator. To install the Jaeger deployment, follow the steps in [Jaeger Operator documentation - Creating a new instance](https://github.com/jaegertracing/helm-charts/tree/master/charts/jaeger-operator#creating-a-new-jaeger-instance) and deploy the preferred Jaeger deployment.

> !!! Note

Reviewer: Is this a sequence of steps? If not, we should have this as bullet points instead of a numbered list.

Author: No. These are some things to note before configuring Jaeger.

Author: Done

>     1. There are some limitations in the Jaeger client, which by default uses a UDP sender as mentioned in [the Jaeger documentation](https://www.jaegertracing.io/docs/1.22/client-libraries/). If the payload size exceeds 65 KB, spans might get lost in the Jaeger console. 

Reviewer: ```suggestion
    1. There are some limitations because the Jaeger client, by default, uses a UDP sender as mentioned in [the Jaeger documentation](https://www.jaegertracing.io/docs/1.22/client-libraries/). If the payload size exceeds 65 KB, spans might get lost in the Jaeger console. 
```

>     1. There are some limitations in the Jaeger client, which by default uses a UDP sender as mentioned in [the Jaeger documentation](https://www.jaegertracing.io/docs/1.22/client-libraries/). If the payload size exceeds 65 KB, spans might get lost in the Jaeger console. 

Reviewer: not sure what is meant by 'spans' here.

Author: @lkokila Can you please comment on this?

Reviewer 2: "Spans' refer to Jaeger terminology. Should be comprehensible in the document's context.  

[1]. https://www.jaegertracing.io/docs/1.22/architecture/#span

    2. Jaeger [sampler types](https://www.jaegertracing.io/docs/1.22/sampling/) can also play a major role in tracing. Depending on the TPS, the sampler type should be carefully chosen.

>     3. In general, before including tracing in production deployments it is essential to look into performance tests and scaling requirements. Refer the [Jaeger performance tuning guide](https://www.jaegertracing.io/docs/1.22/performance-tuning/) for details on how to achieve better performance. 

Reviewer: ```suggestion
    3. Be sure to check the performance tests and scaling requirements before including tracing in production deployments . Refer the [Jaeger performance tuning guide](https://www.jaegertracing.io/docs/1.22/performance-tuning/) for details on how to achieve better performance. 
```
